.classpath

```diff
@@ -6,23 +6,13 @@
 			<attribute name="maven.pomderived" value="true"/>
```

Comment:
configuration files should not be pushed. like - .classpath,  .project and target etc
for that use gitignore